community/server/src/main/java/org/neo4j/server/Bootstrapper.java

```diff
@@ -76,6 +77,7 @@ public static void main( String[] args )
     public int start( File configFile, Pair<String, String> ... configOverrides )
     {
         LogProvider userLogProvider = FormattedLogProvider.withoutRenderingContext().toOutputStream( System.out );
+        System.setOut( NullPrintStream.getInstance() );
```

Comment:
Can we at least throw an exception if someone tries to write on stdout? In this way there is the risk to hide problems, since writing to stdout is simply a noop.